tfx/components/transform/executor.py

```diff
@@ -356,16 +382,13 @@ def _GetCachePath(label, params_dict):
         labels.DATA_VIEW_LABEL:
             data_view_uri,
         labels.ANALYZE_DATA_PATHS_LABEL:
-            io_utils.all_files_pattern(train_data_uri),
+            analyze_data_paths,
         labels.ANALYZE_PATHS_FILE_FORMATS_LABEL:
-            labels.FORMAT_TFRECORD,
-        labels.TRANSFORM_DATA_PATHS_LABEL: [
-            io_utils.all_files_pattern(train_data_uri),
-            io_utils.all_files_pattern(eval_data_uri)
-        ],
-        labels.TRANSFORM_PATHS_FILE_FORMATS_LABEL: [
-            labels.FORMAT_TFRECORD, labels.FORMAT_TFRECORD
-        ],
+            [labels.FORMAT_TFRECORD for _ in range(len(analyze_data_paths))],
```

Comment:
This can be simplified with 
`[labels.FORMAT_TFRECORD] * len(analyze_data_paths)`